drops/azure_edge_extensions_lnm_mq_isa95.json

```diff
@@ -0,0 +1,30 @@
+{
+    "Title": "Azure Edge Extensions AIO ISA-95 LNM and MQ",
+    "Summary": "This is a solution to build an ISA-95 network model on top of Azure IoT Operations utilizing AIO LNM and AIO MQ.",
```

Comment:
Suggestion -> Add some extra wording about what "AIO" and LNM and MQ stands for - Probably try to use full names, as people reading the content maybe they're not familiar

Reply:
agree